game/static/game/css/auth.css

```
@@ -161,3 +161,209 @@ body {
+}
+.toast-title {
+    font-size: 0.88rem;
+    font-weight: 700;
+    letter-spacing: 0.3px;
+    margin-bottom: 3px;
+}
+.toast-message {
+    font-size: 0.82rem;
+    color: #a0a0c0;
+    line-height: 1.4;
+    word-break: break-word;
```

Stylelint (17.11.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)